src/server/pages/index.js

```diff
@@ -4,39 +4,95 @@ import signout from './signout'
 import verifyRequest from './verify-request'
 import error from './error'
 import css from '../../css'
+const merge = require('lodash.merge')
+
+export const defaultTranslations = {
```

Comment:
Think we should move this into its own file like `default-events.js` and `default-callbacks.js` are. I think this could actually be just a `default-translations.json` file as well.

Reply:
Agree, will do.